a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -74,19 +73,22 @@ def _set(self, set_method, c, context, listener):
             except Exception as ex:
                 LOG.exception(ex)
 
-            unbound = any([not bool(x.status) for x in bindings])
+            unbound = not bindings or any([not bool(x.status) for x in bindings])
```

Comment:
This can't be correct. It's `True` if `bindings == None` and `False` if `bindings == []`